frontend/src/pages/Alerts/SessionAlert/SessionAlertPage.tsx

```diff
@@ -689,8 +642,6 @@ const SessionAlertForm = ({
 								name={formStore.names.trackProperties}
 							>
 								<Select
-									onSearch={handleTrackPropertiesSearch}
-									options={trackPropertiesSuggestions}
```

Comment:
Wondering how this component works the data fetching logic removed, but thinking it doesn't really matter because this code is no longer accessible, right? I wasn't able to hit this in my click testing.

Reply:
yeah, I decided to just remove this - the component is unusable without the field suggestions, but will continue to display any track properties that were already selected for these deprecated alerts. The old way of searching was across all field names / values which doesn't scale well with how we've been saving them. Users wanting a user / track property alert should use the new alert form.